examples/angular-cli/src/stories/addons/docs/doc-button/doc-button.stories.ts

```diff
@@ -1,9 +1,8 @@
 import { DocButtonComponent } from './doc-button.component';
 
 export default {
-  title: 'DocButton',
+  title: 'Addons/Docs/DocButton',
```

Comment:
All the other `angular-cli` examples are in `Addon/x`. I think "addons" is better, but I suggest making the switch in a separate PR and updating all the other examples at the same time.